packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -133,9 +134,14 @@ const start = async (zcf, privateArgs) => {
   const instanceToRecords = makeLegacyMap('oracleInstance');
 
   let publishedTimestamp = await E(timer).getCurrentTimestamp();
+  const timerBrand = await E(timer).getTimerBrand();
+  const soon = TimeMath.toRel(0n, timerBrand);
+  // TODO: should the caller provide POLL_INTERVAL as a properly
+  // branded time?
```

Comment:
yep.

Reply:
now that we aren't trying to enforce brands on the *input* to the timer service, this question will be deferred for the future (i.e. callers *should* provide POLL_INTERVAL as a properly branded `RelativeTimeRecord`, but aren't yet required to)